config.json

```diff
@@ -244,7 +244,7 @@
           "functional_programming",
           "generics",
           "lists",
-          "loops"
+          "control_flow_loops"
```

Comment:
Was this an intentional change to list ops?

Reply:
Yes, minesweeper uses loops so I grepped for `loops`, and noticed that this one exercise was different. 

It is also the only exercise that has `generics` or `functional_programming`

Reply:
I don't believe Exercism still uses topics in v3, but okay, I'll keep the change.